lib/compress/index.js

```diff
@@ -6066,7 +6066,7 @@ merge(Compressor.prototype, {
                     }));
                 } else {
                     value = fixed.optimize(compressor);
-                    if (value === fixed) value = fixed.clone(true);
+                    // if (value === fixed) value = fixed.clone(true);
```

Comment:
IDK should I remove this or put TODO for futher investigations

Reply:
I was quite curious why this fixed the problem lol. If there is a clone operation here then there must be a valid reason for it. This is probably not it. Less cloning means more mutations, and more mutations means more bugs.

Reply:
Either way this fixes 3 issues and don't break anything. Also I don't think that this should return cloned value, for example we return same value in others `def_optimize` like `AST_SymbolExport`